learn/advanced/asynchronous_operations.mdx

```diff
@@ -101,6 +101,10 @@ In a protected instance, your API key must have access to all indexes (`"indexes
 
 After creating a task, Meilisearch places it in a queue. Enqueued tasks are processed one at a time, following the order in which they were requested.
 
+<Capsule intent="warning" title="Task queue limit">
+When the task queue reaches its limit (about 10GiB), it will throw a `no_space_left_on_device` error. Users will need to delete tasks using the [delete tasks endpoint](/reference/api/tasks#delete-tasks) to continue write operations.
```

Comment:
On the issue @curquiza wrote that 

> v1.1.1 (coming soon) will provide a temporary fix for the task DB issue

So I'm wondering if this note is still necessary.

Reply:
The error message is the temporary fix, if I understand correctly

Reply:
Yes it's a temporary fix until v1.2 is out 😊 
You will be able to remove this sentence once v1.2 is out